fbchat/graphql.py

```diff
@@ -191,28 +191,22 @@ def graphql_to_extensible_attachment(a):
                     description=story["description"].get("text")
                     if story.get("description")
                     else None,
-                    source=story["source"]["text"],
-                    image_url=story["media"]["image"]["uri"]
-                    if story.get("media")
-                    else None,
-                    original_image_url=(
-                        get_url_parameter(story["media"]["image"]["uri"], "url")
-                        if "/safe_image.php" in story["media"]["image"]["uri"]
-                        else story["media"]["image"]["uri"]
-                    )
-                    if story.get("media")
-                    else None,
-                    image_width=story["media"]["image"]["width"]
-                    if story.get("media")
-                    else None,
-                    image_height=story["media"]["image"]["height"]
-                    if story.get("media")
-                    else None,
+                    source=story["source"].get("text"),
                     attachments=[
                         graphql_to_subattachment(attachment)
                         for attachment in story.get("subattachments")
                     ],
                 )
+                if story.get("media"):
+                    if story["media"].get("image"):
```

Comment:
Merge the two if-statements with `and`

Reply:
Are sure this won't throw an error if `media` will be `None`? I tried this in Python console and it through an error.

Reply:
```py
>>> story = {}
>>> story.get('media') and story["media"].get("image")
None
```
Yes, I'm sure 😁. The expression behind `and` won't get executed unless the previous expression was `True`. The same goes for `or`. Another example:
```py
>>> False and invalid
False
>>> True or invalid
True
```